src/module.c

```diff
@@ -8940,7 +8940,14 @@ void moduleCallClusterReceivers(const char *sender_id,
  * was already a registered callback, this will replace the callback function
  * with the one provided, otherwise if the callback is set to NULL and there
  * is already a callback for this function, the callback is unregistered
- * (so this API call is also used in order to delete the receiver). */
+ * (so this API call is also used in order to delete the receiver).
+ *
+ * When a message of this type is received, the registered callback function
+ * will be invoked with details, including the node ID.
+ *
+ * Behavior Change:
+ * Before Valkey 8.1: The node ID was not null-terminated.
+ * Currently: The node ID is now null-terminated for easier handling. */
```

Comment:
Yes, great! Just some minor suggestions:

* Mention that it's 40 bytes long, so it's possible know how to write modules to work with older Valkey versions.
* Write "In Valkey 8.1 and later ..." like how write in some other places in the docs.

```suggestion
 * will be invoked with details, including the 40 bytes node ID of the sender.
 *
 * In Valkey 8.1 and later, the node ID is null-terminated. Prior to 8.1, it was
 * not null-terminated.
```